### PR TITLE
update upgrade scripts

### LINKED ISSUE #2587: update upgrade scripts
just make sure the 3.0 upgrade script renames everything that's been renamed.

### PR DESCRIPTION
**Pull request by giladgray**
#### Changes proposed in this pull request:

add fixes for latest changes to 3.0.0 upgrade script.

### CODE REVIEW

File: packages/core/scripts/upgrade-blueprint-3.0.0-rename.sh [hunk 1]
```diff
@@ -165,7 +165,30 @@ echo "
...
+renameString 'requiredLabel={true}' 'labelInfo="(required)"'
+renameProp requiredLabel labelInfo
+
+# Classes constants
+renameString "Classes\.CALLOUT_TITLE" "Classes.HEADING"
+renameString "Classes\.DIALOG_TITLE" "Classes.HEADING"
+renameString "Classes\.HOTKEY_GROUP" "Classes.HEADING"
+renameString "Classes\.NON_IDEAL_STATE_TITLE" "Classes.HEADING"
+renameString "Classes\.UI_TEXT_LARGE" "Classes.UI_TEXT, Classes.TEXT_LARGE"
+renameString "Classes\.RUNNING_TEXT" "Classes.RUNNING_TEXT, Classes.TEXT_LARGE"
```
Comment (invliD) [inline packages/core/scripts/upgrade-blueprint-3.0.0-rename.sh]: Why should running text now be large?

Comment (giladgray) [reply packages/core/scripts/upgrade-blueprint-3.0.0-rename.sh]: the default used to be large and it had a `small` modifier. i standardized it in #2366

Comment (giladgray) [reply packages/core/scripts/upgrade-blueprint-3.0.0-rename.sh]: we used to have `RUNNING_TEXT` and [`RUNNING_TEXT_SMALL`](https://github.com/palantir/blueprint/pull/2366/files#diff-729752f23dd9738de732ea84260397beL49) where the regular one was actually large font size, but in #2366 i added a new `TEXT_LARGE` modifier, so `RUNNING_TEXT` in 3.0 uses default normal font size. 

this means that folks who were using `RUNNING_TEXT` in 1.x should now also apply `TEXT_LARGE` in 3.0.

**Comment by giladgray** [comment]
`HTMLTable`

**Pull request merged**